www/webkit2-gtk/Portfile

```diff
@@ -184,7 +204,7 @@ if {${os.platform} eq "darwin" && ${os.major} < 13} {
     # contents of snowmath.h should someday become part of gcc6 cmath
     if { ${configure.cxx_stdlib} eq "macports-libstdc++"  && ${os.major} < 11 } {
         configure.cxxflags-append -D_GLIBCXX_USE_C99_MATH_TR1=1
-        configure.cxxflags-append -include ${filespath}/snowmath.h
+        configure.cxxflags-append -include ${portpath}/${filesdir}/snowmath.h
```

Comment:
Why did you change this? It looks better the way it was.

Reply:
Hi @kencu, could you have a look at this? This change is also stolen from your repo.

Reply:
I think the `${portpath}/${filesdir}` thing was my original go at this, and then Ryan fixed it up to be ${filespath}, but I never changed it in my own repo version.

Reply:
Thanks I didn't notice that. Reverted in the latest push.